realtime.go

```diff
@@ -0,0 +1,83 @@
+package flagsmith
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Flagsmith/flagsmith-go-client/v4/flagengine/environments"
+)
+
+func (c *Client) startRealtimeUpdates(ctx context.Context) {
+	err := c.UpdateEnvironment(ctx)
+	if err != nil {
+		panic("Failed to fetch the environment while configuring real-time updates")
+	}
+	env, _ := c.environment.Load().(*environments.EnvironmentModel)
+	stream_url := c.config.realtimeBaseUrl + "sse/environments/" + env.APIKey + "/stream"
+	envUpdatedAt := env.UpdatedAt
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		default:
+			resp, err := http.Get(stream_url)
+			if err != nil {
+				c.log.Errorf("Error connecting to realtime server: %v", err)
+				continue
+			}
+			defer resp.Body.Close()
+
+			scanner := bufio.NewScanner(resp.Body)
+			for scanner.Scan() {
+				line := scanner.Text()
+				if strings.HasPrefix(line, "data: ") {
+					parsedTime, err := parseUpdatedAtFromSSE(line)
+					if err != nil {
+						c.log.Errorf("Error reading realtime stream: %v", err)
+						return
+					}
```

Comment:
Are you sure you want to `return` here? Shouldn't we go back to listening to the stream on the hopes of the next data line includes the `updated_at` field?

Reply:
Yeah, I think you are right. I will update